tests/test_models.py

```diff
@@ -12,10 +12,11 @@
 #
 #  You should have received a copy of the GNU Affero General Public License
 #  along with this program.  If not, see <https://www.gnu.org/licenses/>.
-from chalicelib.models import ModuleName, ModuleNameAttribute
 
 
 def test_module_name_init():
+    from chalicelib.models import ModuleName
```

Comment:
This change was put so we had chance to configure the location of the PynamoDB table.